### PR TITLE
go.mod: Update proto dependecies.

### PR DESCRIPTION
**Pull request by dilyevsky**
Resolves annoying stderr prints (fixed in golang/protobuf upstream) such as:
```
2019/12/17 15:44:46 proto: tag has too few fields: "-"
2019/12/17 15:44:46 proto: don't know how to copy 0
2019/12/17 15:44:46 proto: don't know how to copy 0
```

Also bump gogo/protobuf. Run `go mod tidy`

---
This change is [Reviewable](https://reviewable.io/reviews/stripe/skycfg/67)

### CODE REVIEW

**Comment by dilyevsky** [comment]
I think it's mostly when trying to merge nil fields (frequent in Kubernetes psuedo-protos) for the first one and from fields that are missing `protobuf` tag (such as https://github.com/kubernetes/api/blob/63b8484e576699c2f714dd6319c827a568a95090/apps/v1beta1/types.go#L62) for the rest of them

**Comment by dilyevsky** [comment]
Oh actually it's this - https://github.com/kubernetes/apimachinery/blob/b789a6e6b31f49e34cd0b0876791677a85bbb375/pkg/apis/meta/v1/time.go#L34

The prints are still there btw they just don't crap into your stderr

File: go.mod [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+	github.com/golang/protobuf v1.3.2
 	github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348
 	go.starlark.net v0.0.0-20190604130855-6ddc71c0ba77
-	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6 // indirect
 	gopkg.in/yaml.v2 v2.2.1
 )
 
 replace github.com/kylelemons/godebug => github.com/jmillikin-stripe/godebug v0.0.0-20180620173319-8279e1966bc1
+
+go 1.13
```
Comment (jmillikin-stripe) [inline go.mod]: It looks like this line is causing the CI build failure. Can you remove it, and also revert the change to `.travis.yml`?

Comment (dilyevsky) [reply go.mod]: Yeah I was in the middle of making another PR for this when I saw your other change. Done.

**Comment by dilyevsky** [comment]
Passing now. PTAL

**Pull request merged**